osu.Game/Overlays/Toolbar/Toolbar.cs

```diff
@@ -79,7 +79,7 @@ private void load(OsuGame osuGame, Bindable<RulesetInfo> parentRuleset)
                         //{
                         //    Icon = FontAwesome.Solid.search
                         //},
-                        userButton = new ToolbarUserButton(),
+                        userButton = new ToolbarUserButton(this),
```

Comment:
What is the meaning of these changes? You can't be doing things like this in a commit with the message of `Updated .gitignore`.

Reply:
Oh, i started working on an other issue, and i've tried something, but i added it in my commit gitignore, i will change it rn